src/flows/Upload/CreateGameVersionStep.tsx

```diff
@@ -37,57 +38,60 @@ export function CreateGameVersionStep({
   }, [game?.gameVersions.nodes])
 
   useEffect(() => {
-    if (typeof challengeSupport === "boolean") {
-      const zip = new AdmZip()
-      const gameFiles = getGameFiles(gameDir)
+    if (typeof challengeSupport === "boolean" || multiplayer) {
```

Comment:
I am unable to follow the logic inside this file, when does this effect execute? Why this if is written like that?

Reply:
hopefully this makes it easier to follow until we have some kind of bigger refactoring https://github.com/rune/rune-games-cli/pull/32/commits/a1b5a3ad4ccd3f7709995d6f6809e3833b1474ef